apps/indexer/src/api/repositories/drizzle.ts

```diff
@@ -156,6 +175,136 @@ export class DrizzleRepository {
     return db.$count(proposalsOnchain);
   }
 
+  async getProposalNonVoters(
+    proposalId: string,
+    skip: number,
+    limit: number,
+    orderDirection: "asc" | "desc",
+    addresses?: Address[],
+  ): Promise<{ voter: Address; votingPower: bigint }[]> {
+    return await db
+      .select({
+        voter: accountPower.accountId,
+        votingPower: accountPower.votingPower,
+      })
+      .from(accountPower)
```

Comment:
This account power is time sensitive? I mean, the current account power is being used right? But a "possible voter" needs to be some one that has voting power in the "timestamp" of the proposal. Isnt it? Like, if i had 0_vp in the beginning of the prop, but now i have  1k_vp, im not a "possible voter" right?